src/runtime_src/core/pcie/linux/pcidrv.cpp

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+#include <filesystem>
```

Comment:
I don't think std::filesystem works on Ubuntu18.04?

Reply:
Indeed! Isn't it part of standard C++17? I'm confused. I'll change it back. Sigh...

Reply:
The compile on Ubuntu18.04 is not C++17 std compliant for at least std::filesystem.   We need to get rid of this, it is the hold out for general move from boost::filesystem -> std::filesystem

Reply:
I have been trying not to make too much change to the existing code which is not really necessary. But, this one was too attempting, hehe...

Reply:
Who would use a 4+ year old system anyway? :-)
Just add a more modern compiler to the install dependency?
According to https://en.cppreference.com/w/cpp/compiler_support/17#C.2B.2B17_library_features this is implemented in `g++-8`, `clang++-7` and above.
Ubuntu 18.04 might come with an old `g++` but you could install `g++-8` and/or `clang++-10` package on it, with their C++ standard libraries.
https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=gcc-8&searchon=names